eth/backend.go

```diff
@@ -641,9 +642,15 @@ func New(stack *node.Node, config *ethconfig.Config, logger log.Logger) (*Ethere
 		if currentBlock.BaseFee() != nil {
 			baseFee = currentBlock.BaseFee().Uint64()
 		}
+		blobFee := uint64(1)
+		if currentBlock.Header().ExcessBlobGas != nil {
+			b, err := misc.GetBlobGasPrice(misc.CalcExcessBlobGas(currentBlock.Header()))
+			if err == nil && b.Cmp(uint256.NewInt(0)) > 0 {
+				blobFee = b.Uint64()
+			}
+		}
 		backend.notifications.Accumulator.StartChange(currentBlock.NumberU64(), currentBlock.Hash(), nil, false)
-		backend.notifications.Accumulator.SendAndReset(ctx, backend.notifications.StateChangesConsumer, baseFee, currentBlock.GasLimit(), 0)
-
+		backend.notifications.Accumulator.SendAndReset(ctx, backend.notifications.StateChangesConsumer, baseFee, blobFee, currentBlock.GasLimit(), 0)
```

Comment:
If we are sending pending rather than current blobFee, it makes sense to do the same for baseFee (pending instead of current).

Reply:
Yes, i will make that change as well. Meanwhile, in this case we are sending values (to txPool) with respect to the CurrentBlock, which is where the sync is at. Should we consider sending values for the canonical head instead?